src/components/backend-ai-storage-list.ts

```diff
@@ -1339,8 +1339,12 @@ export default class BackendAiStorageList extends BackendAIPage {
       let temp: any = [];
       for (let i = 0; i < e.dataTransfer.files.length; i++) {
         const file = e.dataTransfer.files[i];
+        /* Drag & Drop file upload size limits to 1MiB */
         if (file.size > 2 ** 20) {
-          console.log('File size limit (< 1 MiB)');
+          this.notification.text = _text('data.explorer.DragDropFileUploadSizeLimit');
+          this.notification.show();
+          return;
+          // console.log('File size limit (< 1 MiB)');
```

Comment:
In my opinion, 1 MB is too small. How about increasing it to 10 MB or 50 MB, perhaps?